AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/Model/KeyStore.smithy

```diff
@@ -88,19 +84,70 @@ structure KeyStoreConfig {
   //# 
   //# - [ID](#keystore-id)
   //# - [AWS KMS Grant Tokens](#aws-kms-grant-tokens)
+  //# - [Storage](#storage)
   //# - [DynamoDb Client](#dynamodb-client)
+  //# - [Table Name](#table-name)
   //# - [KMS Client](#kms-client)
-  
+
+  @javadoc("The key management configuration for this Key Store.")
+  keyManagement: KeyManagement,
+
+  @javadoc("The DynamoDB table name that backs this Key Store.")
+  ddbTableName: TableName,
```

Comment:
Blocking: By removing the `@required`, the type for `ddbTableName` has changed from String to Optional String.

In Dafny Source code, this a breaking change...
I am not sure if it is a breaking change in our compiled code...